### DISCUSSION TITLE
Play only specific audio present in a channel

### DISCUSSION THREAD

**Discussion by AravindJ441** [katspaugh/wavesurfer.js#3880] [Q&A]
Hi, i'm using   "wavesurfer.js": "^7.8.0" in angular  and  for the recordings
i am getting 2 channels and is there any way to get audio  data from channel 1

**Comment by katspaugh** [comment]
You can use a stereo panner to play only one channel. See an example here: https://wavesurfer.xyz/examples/?4f00b61846e765f6d68a5c56c9e21379
